docs/package/gluon-config-mode-geo-location.rst

```diff
@@ -0,0 +1,55 @@
+gluon-config-mode-geo-location
+==============================
+
+This package allows the user to set latitude, longitude and optionally altitude
+within config mode. There are 2 types of this package.
```

Comment:
> I suppose it is not possible to merge the two packages and have a site.conf option decide whether to enable the feature, because the site.conf values are not available in the Makefile, only environment variables will be, correct?

I am not sure what do you mean. There are two completly seperated packages where you need to add to the site.mk. Depending on which package you have put into the site.mk you will have optional additional parameters in the site.conf

> The package is split off to prevent cluttering the filesystems with all required js/css files, when the map is not wanted, correct?

Its split off, to give the communities different options. Maybe some communities don't want to have a map in the config-mode. Therefore it is not necessary to have dead code on the Routers flash.